evm-schema/src/lib.rs

```diff
@@ -68,8 +68,6 @@ pub fn transactions_schema() -> Schema {
         Field::new("transaction_index", DataType::UInt64, true),
         Field::new("value", DataType::Decimal256(76, 0), true),
         Field::new("v", DataType::UInt8, true),
-        // keep these binary even though they are uint256 because they don't fit in i256 in
```

Comment:
should keep comment imo because someone looking at this might be confused. Everything is typed Decimal256 but these fields are binary